docs/guided_tour_summary_pages_requests.md

```diff
@@ -0,0 +1,136 @@
+
+```
+SELECT url pageUrl, 
+       reqImg, 
+       bytesImg
+FROM `httparchive.summary_pages.2018_09_01_desktop` 
+ORDER BY bytesImg DESC
+```
+![example results](./images/guided_tour_summary_requests_pages_join_example1.jpg)
+
+
+In the previous example we didn't use the requests table at all. However if we wanted to, we could do the same analysis by JOINing the request table. In order to do this, I've made the following modifications to the query above:
+* Named the tables (ie, so we can reference "httparchive.summary_pages.2018_09_01_desktop" as "pages") 
+* Reference the tables for some of the columns that exist in both tables (such as pages.url)
+* Since we're focuing on images, we'll include `requests.type="image"` in our WHERE clause. 
+* JOIN the tables `ON pages.pageid = requests.pageid`
+The below query is identifical to the previous one, but uses both tables.
+
+```
+SELECT pages.url pageUrl,
+       count(*) reqImg, 
+       SUM(respBodySize) bytesImg 
+FROM `httparchive.summary_pages.2018_09_01_desktop`  pages
+INNER JOIN `httparchive.summary_requests.2018_09_01_desktop`  requests
+ON pages.pageid = requests.pageid
+WHERE requests.type = "image"
+GROUP BY pageUrl
+ORDER BY bytesImg DESC
+```
+![example results](./images/guided_tour_summary_requests_pages_join_example2.jpg)
+
+Some of these image weights seems quite extreme, but these represent the most image heavy sites in the archive. If you want to validate this, you can load one of the pages view DevTools or WebPageTest and see for yourself.  For example based on the results above,  http://thaliahallchicago.com/ has 80 images totaling 117MB. WebPageTest confirms that this is accurate (although the image weight likley changed slightly between when the HTTP Archive saw the page vs when this manual test was performed.
+
+![example results](./images/guided_tour_summary_requests_pages_join_example2_wpt.jpg)
+
+If this was all the information we needed, then JOINING the table would have been unecessary. However we want to look at the relationship between the largest image and the overall image weight. In the query below you can add added `MAX(responseBodySize)` to look for the size of the largest image.
+
+```
+SELECT pages.url pageUrl,
+       count(*) reqImg, 
+       SUM(respBodySize) bytesImg, 
+       MAX(respBodySize) largestImg 
+FROM `httparchive.summary_pages.2018_09_01_desktop`  pages
+INNER JOIN `httparchive.summary_requests.2018_09_01_desktop`  requests
+ON pages.pageid = requests.pageid
+WHERE requests.type = "image"
+GROUP BY pageUrl
+ORDER BY bytesImg DESC
+```
+![example results](./images/guided_tour_summary_requests_pages_join_example3.jpg)
+
+Now let's add a simple formula to look at the relationship of the largest image on a page to the image weight:  largestImg/bytesImg. However seen we can't reference the aggregate names in the query, we'll include the function calls. `MAX(respBodySize)/SUM(respBodySize)`.

```

Comment:
```suggestion
Now let's add a simple formula to look at the relationship of the largest image on a page to the image weight:  largestImg/bytesImg. However since we can't reference the aggregate names in the query, we'll include the function calls. `MAX(respBodySize)/SUM(respBodySize)`.
```